JSDemos/Demos/DataGrid/CustomSummaries/description.md

```diff
@@ -1 +1,13 @@
-The DataGrid support custom summary calculation. In&nbsp;this example, summaries are only calculated against selected grid rows. This behavior is&nbsp;implemented using the [calculateCustomSummary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/#calculateCustomSummary) and [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) properties.
\ No newline at end of file
+DataGrid can apply custom server-side and client-side functions to calculate total and group summaries. If the dataset is small, you should choose the client-side implementation.
+
+This demo shows how to add such a summary: it calculates the sum of sale amounts for all selected rows. To achieve this behavior, we have done the following:
```

Comment:
You're describing this particular use case with selection, while it is better to describe things in general and then say how they relate to the use case shown in the demo.